jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -548,8 +548,8 @@ void java_bytecode_parsert::get_class_refs()
 
     case CONSTANT_NameAndType:
       {
-        typet t=java_type_from_string(id2string(pool_entry(c.ref2).s));
-        get_class_refs_rec(t);
+        const auto t = java_type_from_string(id2string(pool_entry(c.ref2).s));
```

Comment:
:pick: while in the area, `t` -> `type`

Reply:
I just got rid of it, it didn't carry much value...